code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -120,6 +120,71 @@
 	mute = MUTE_NONE
 	security_lock = TRUE
 	locked = FALSE
+	materials = list(MAT_METAL=500, MAT_GLASS=50)
+
+/obj/item/clothing/mask/muzzle/safety/shock
+	name = "shock muzzle"
+	desc = "A muzzle designed to prevent biting.  This one is fitted with a behavior correction system."
+	var/obj/item/assembly/trigger = null
+	origin_tech = "materials=1;engineering=1"
+	materials = list(MAT_METAL=500, MAT_GLASS=50)
+
+/obj/item/clothing/mask/muzzle/safety/shock/attackby(obj/item/W, mob/user, params)
+	if(isscrewdriver(W))
+		to_chat(user, "<span class='notice'>You disassemble [src].</span>")
+		trigger.forceMove(get_turf(user))
+		trigger.master = null
+		trigger.holder = null
+		trigger = null
+	else if(trigger)
+		to_chat(user, "<span class='notice'>Something is already attached to [src].</span>")
+	else if(istype(W, /obj/item/assembly/signaler) || istype(W, /obj/item/assembly/voice))
+		user.drop_item()
+		trigger = W
+		trigger.forceMove(src)
+		trigger.master = src
+		trigger.holder = src
+		to_chat(user, "<span class='notice'>You attach the [W] to [src].</span>")
+	else
+		return ..()
+
+
+/obj/item/clothing/mask/muzzle/safety/shock/equipped(obj/item/clothing/C)

```

Comment:
This is not how the `equipped` proc works.
[It's used to fire various events that you want to have happen when the item is worn by a mob](https://github.com/ParadiseSS13/Paradise/blob/a20ce9b7ced9563f801f2a0b1abcbbb24e50da83/code/game/objects/items.dm#L350-L354)

Reply:
Made a unique proc instead of overloading equipped.